cookbooks/aws-parallelcluster-config/files/default/cloudwatch_agent/cloudwatch_agent_config.json

```diff
@@ -3,7 +3,8 @@
     "month_first": "%b %-d %H:%M:%S",
     "default": "%Y-%m-%d %H:%M:%S,%f",
     "bracket_default": "[%Y-%m-%d %H:%M:%S]",
-    "slurm": "%Y-%m-%dT%H:%M:%S.%f"
+    "slurm": "%Y-%m-%dT%H:%M:%S.%f",
+    "metric": ""
```

Comment:
Minor question for the name "metric" here, does it make sense to name it as "empty", or "ignore", "none", "skip", etc., so that we can re-use it in the future for any situation (not limited to the metric use case) when we don't want a timestamp?

Reply:
I think you're right that "metric" is wrong. I think the correct name should be "json" since you should only not have a timestamp in the case where you want CW to parse JSON logs.